R/executeDqChecks.R

```diff
@@ -252,6 +253,10 @@ executeDqChecks <- function(connectionDetails,
 
   if (length(checkNames) > 0) {
     checkDescriptionsDf <- checkDescriptionsDf[checkDescriptionsDf$checkName %in% checkNames, ]
+  } else {
+    # No checkNames specified. Do not run plausibleDuringLife and plausibleTemportalAfter as these are superseded three new checks
```

Comment:
@dimshitc This disables the old checks, if user did not specify them specifically. We do need to document this behaviour.

Reply:
This makes sense to me.

Should we set threshold in all these checks to 0? as these are obvious mistakes and shouldn't be in the data.

Reply:
Yes, that makes even more sense. I added the following warnings if one of the old checks is run:
`"DEPRECATION WARNING - The plausibleDuringLife check has been reimplemented with the plausibleBeforeDeath check."`

`DEPRECATION WARNING - The plausibleTemporalAfter check has been reimplemented with the plausibleAfterBirth and plausibleStartBeforeEnd checks.`

Reply:
Regarding disabling the checks by the user; there is not really a nice solution as `executeDqChecks` only takes the names of checks to be run, not an argument with checks to exclude. So the user needs to list all checks except these two. Or does one of you see a more elegant solution?

Reply:
Warnings look great; thanks!  

And I wouldn't call it elegant, but I did add these instructions to the docs earlier this year on how to exclude a list of checks: https://github.com/OHDSI/DataQualityDashboard/blob/81230bde213b99c5d26a177c7922b70a15a06349/vignettes/DataQualityDashboard.rmd#L114-L120

I think for now we can point to this in the release notes.  In the future I'd love to improve the workflow for enabling/disabling checks and editing thresholds (a la #253 ).